app/serializers/protocol_serializer.rb

```diff
@@ -85,12 +85,13 @@ def in_repository
     !object.in_module?
   end
 
-  def disabled_drafting
-    return false unless can_create_protocols_in_repository?(object.team)
+  # rubocop:disable Naming/PredicateName
+  def has_draft
```

Comment:
Probably should be called `has_draft?`, since we know it always returns a boolean.